dropwizard-jersey/src/main/java/io/dropwizard/jersey/jackson/JsonProcessingExceptionMapper.java

```diff
@@ -38,12 +38,14 @@ public Response toResponse(JsonProcessingException exception) {
             return super.toResponse(exception); // LoggingExceptionMapper will log exception
         }
 
-        final String message = exception.getOriginalMessage();
-
         /*
          * Otherwise, it's those pesky users.
          */
-        LOGGER.debug("Unable to process JSON", exception);
+        if (LOGGER.isDebugEnabled()) {
```

Comment:
It looks to me, since nothing fancy is going on with the log format, `isDebugEnabled` is not necessary, considering that `debug` calls `isDebugEnabled` as well before actually logging.